papis/commands/add.py

```diff
@@ -417,8 +418,17 @@ def run(paths: List[str],
 
         if link:
             in_file_abspath = os.path.abspath(in_file_path)
-            logger.info("[SYMLINK] '%s' to '%s'.", in_file_abspath, tmp_end_filepath)
-            os.symlink(in_file_abspath, tmp_end_filepath)
+            try:
+                logger.info("[SYMLINK] '%s' to '%s'.", in_file_abspath,
+                            tmp_end_filepath)
+                os.symlink(in_file_abspath, tmp_end_filepath)
+            except OSError as e:
+                missing_privilege_error = 1314
+                if sys.platform == "win32" and e.winerror == missing_privilege_error:
+                    logger.error("Failed to link due to insufficient permissions.  You "
+                                 "can try again after enabling the 'Developer mode' "
+                                 "and restarting.")
+                raise
```

Comment:
Does the exception not contain any useful information here? We usually do
```
logger.error(..., exc_info=exc)
```
to show the exception message or a traceback.

Reply:
I can't check, but does this throw just a generic `OSError`? Maybe we can better check for a `PermissionError` or something to show that message.